### PR TITLE
Project Auth - Frida & Emma

### PR DESCRIPTION
**Pull request by fridamaria**
Link to my deployed project:
Backend: https://fridamaria-secret.herokuapp.com/
Frontend: https://fridamaria-secret.netlify.app/

My reflections on how this week's project turned out:
The project turned out well, even though we had some troubles with the error handling. Pairing worked great and we both took turn writing code using vscode live share.

Things I'd like to have clarified or explained in more detail:
We did not manage to get the error/status messages from the backend to show up in our frontend. We tried reading ut from the json but only got undefined. Would like to have some clarification regarding best practice.

### CODE REVIEW

**Comment by fridamaria** [comment]
Edit: we managed to get the backend messages to show up in the frontend when signing up

File: frontend/src/pages/SignIn.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        body: JSON.stringify(signInValues)
+      }
+    )
+
+      .then(res => {
+        if (!res.ok) {
+          throw new Error('Unable to sign in.')
+        }
+        res.json().then(data => {
+          if (data.notFound !== true) {
```
Comment (HIPPIEKICK) [inline frontend/src/pages/SignIn.js:91]: This is a little hard to read, with the double negatives. Is it possible to write this as a positive statement instead?

File: frontend/src/pages/SecretPage.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+export const SecretPage = () => {
+  const url = 'https://fridamaria-secret.herokuapp.com/secrets'
+  const [message, setMessage] = useState('')
+  const [error, setError] = useState('')
+  const history = useHistory()
+
+  const handleSignOut = () => {
+    localStorage.removeItem('accessToken')
+    history.push('/')
+  }
```
Comment (HIPPIEKICK) [inline frontend/src/pages/SecretPage.js:61]: 🤩

File: frontend/src/pages/SignUp.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            Name
+        <input type='text' value={signUpValues.name} onChange={event => setSignUpValues({ ...signUpValues, name: event.target.value })} />
+          </Label>
+          <Label>
+            E-mail
+        <input type='email' required value={signUpValues.email} onChange={event => setSignUpValues({ ...signUpValues, email: event.target.value })} />
+          </Label>
+          <Label>
+            Password
+        <input type='password' required minLength='4' value={signUpValues.password} onChange={event => setSignUpValues({ ...signUpValues, password: event.target.value })} />
```
Comment (HIPPIEKICK) [inline frontend/src/pages/SignUp.js:111]: Good that you're checking this here as well!

**Pull request closed without merging**